src/feb.c

```diff
@@ -1464,6 +1468,31 @@ int API_FUNC qthread_readFE(aligned_t *restrict       dest,
     return QTHREAD_SUCCESS;
 }                      /*}}} */
 
+/* the way this works is that:
+ * 1 - src's FEB state is ignored
+ * 2 - data is copied from src to destination
+ */
+
+int API_FUNC qthread_readXX(aligned_t *restrict       dest,
+                            const aligned_t *restrict src)
+{                      /*{{{ */
+    qthread_t          *me      = qthread_internal_self();
+
+    assert(qthread_library_initialized);
+
+    if (!me) {
+        return qthread_feb_blocker_func(dest, (void *)src, READXX);
```

Comment:
I considered #ifdefing the blocker func call if debugging or FEB profiling was turned on, but decided against it, because in the presence of other FEB operations, doing a blocker func seemed relatively cheap. i.e. who cares if readxx is a little more expensive considering the cost of other FEB operations that are likely going to occur in the same program

But I think you're right that if we don't need `QTHREAD_FEB_UNIQUERECORD`, then there's no reason to include the shep id in the debug code, so I'll just remove all that